iac/main/resources/sustainability.yml

```diff
@@ -21,6 +21,20 @@ SustainabilityBucket:
           ExpirationInDays: 30
           NoncurrentVersionExpiration:
             NoncurrentDays: 10
+    ReplicationConfiguration:
+      Role: !GetAtt SustainabilityBucketRole.Arn
+      Rules:
+        - Id: SustainabilityBucketRule
+          Status: Enabled
+          Priority: 1
+          DeleteMarkerReplication:
+            Status: Enabled
+          Destination:
+            Bucket: !Sub 'arn:aws:s3:::production-dap-sustainability-921370741319-shared'
+            Metrics:
+              Status: Enabled
+          Filter:
+            Prefix: ''
```

Comment:
is the Filter needed if it's just an empty Prefix?

Reply:
seems we need these for backwards compatibility: https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-add-config.html#replication-config-min-rule-config